src/components/atoms/TitleSeparator.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <h5 class="right">
+    <span><slot></slot></span>
+  </h5>
+</template>
+<style scoped>
+h5 {
+  color: #fff;
+  display: flex;
+  align-items: center;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  font-family: Montserrat, sans-serif;
+  color: #fff;
+  font-size: 14px;
+  line-height: 18px;
+  font-weight: 700;
+  letter-spacing: 1.12px;
+  text-transform: uppercase;
```

Comment:
Please delete any css related to typography and use the global typography class `text-h5` or mixin `t-h5()`

Reply:
Actually...
When checking the designs it seems the TitleSeparator will need to be a different typography class based on the window width.

So it might be interesting that you provide a "size" prop with possible values: "md" and "sm" and have the typography class be chosen automatically based on that prop.

<img width="702" alt="image" src="https://user-images.githubusercontent.com/3253920/94399156-d2a01100-01a1-11eb-98ad-b1a44e1a307c.png">